.github/workflows/build.yml

```diff
@@ -5,9 +5,9 @@ name: Build
 
 on:
   push:
-    branches: [ main ]
```

Comment:
Can we use '**' as described [here](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet) to trigger the actions for all branches?

Reply:
Why do we need so many branches?

Reply:
That makes it possible for our feature-branches to be tested too. This enables having tested the changes before we merge them.